src/main/java/uk/ac/ebi/ena/webin/cli/WebinCliCommand.java

```diff
@@ -130,7 +130,9 @@ public interface Descriptions {
         String centerName = "Mandatory center name for broker accounts.";
         String submit = "Validate, upload and submit files.";
         String validate = "Validate files without uploading or submitting them.";
-        String quick = "Validates submitted read files (BAM, CRAM, Fastq) within a fixed time period (5 minutes). All CRAM reference sequence md5 checksums are always validated. When this option is used files may only be partially validated and may fail post-submission processing.";
+        String quick = "By default, validate submitted BAM & CRAM read files within a fixed time period (5 minutes) " +
```

Comment:
I think it might be best just to remove the -quick option (and therefore the quick description) as we are releasing a new major version and can break backward compatibility (not that many people would have used the -quick option).